bundles/org.openhab.binding.gree/src/main/java/org/openhab/binding/gree/internal/handler/GreeHandler.java

```diff
@@ -84,7 +84,7 @@ public GreeHandler(Thing thing, GreeTranslationProvider messages, GreeDeviceFind
     @Override
     public void initialize() {
         config = getConfigAs(GreeConfiguration.class);
-        if (config.ipAddress.isEmpty() || (config.refresh < 0)) {
+        if (config.ipAddress.isEmpty() || (config.refreshInterval < 0)) {
```

Comment:
While at it the check can also be improved:
```suggestion
        if (config.ipAddress.isBlank() || (config.refreshInterval < 0)) {
```